replica/sstables_adder.hh

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2014-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+#pragma once
+
+#include <vector>
+
+#include <seastar/util/bool_class.hh>
+
+#include "seastarx.hh"
+#include "row_cache.hh"
```

Comment:
You're adding a heavy header where it's not necessary, this breaks modularization and causes excessive recompilation.

Reply:
Well, it is needed here.

Reply:
The most accurate approach is to extract external_updater into a separate header and include only that. However, it's overkill in this case.